test/util/validator.test.js

```diff
@@ -12,43 +12,59 @@ test('validateInputJson', () => {
   }
 
   const stringFields = [ 'currencyCode', 'partnerId', 'accessKey', 'secretKey' ]
```

Comment:
Move to strings test

Reply:
The stringFields array is used by multiple tests.